tests/e2e/tools/lib/editor.js

```diff
@@ -1,3 +1,5 @@
+import {expect, Locator} from '@playwright/test'; // eslint-disable-line no-unused-vars
```

Comment:
Why do we need to import `Locator`? I don't see it used

Reply:
It's used [here](https://github.com/greenpeace/planet4-master-theme/blob/6dafd13dbdc1a92c7f1bcaa4a4b39d53b66d0e51/tests/e2e/tools/lib/editor.js#L6). It's going to be required when we'll update webpack and related deps. Eslint is going to be more strict.

Reply:
I am not going to fix the specs on tha PR, just make them pass and finally make sure that Node 18 is potentially ready to be merged.